kale/embed/mpca.py

```diff
@@ -146,23 +149,23 @@ def _fit(self, X):
         for i in range(self.ndim - 1):
             eig_vals, eig_vecs = np.linalg.eig(Phi[i])
             idx_sorted = eig_vals.argsort()[::-1]
-            Us[i] = eig_vecs[:, idx_sorted]
+            Vs[i] = eig_vecs[:, idx_sorted]
```

Comment:
We can use a more interpretable name for `Vs`, e.g. `sorted_eig_vecs` or something else more readable.

Reply:
will do